src/bun.js/test/jest.zig

```diff
@@ -1216,6 +1248,62 @@ pub const Expect = struct {
         return .zero;
     }
 
+    pub fn toMatchSnapshot(this: *Expect, globalObject: *JSC.JSGlobalObject, callframe: *JSC.CallFrame) callconv(.C) JSValue {
+        defer this.postMatch(globalObject);
+
+        const _arguments = callframe.arguments(1);
+        const arguments: []const JSValue = _arguments.ptr[0.._arguments.len];
+        var hint: ?JSC.JSValue = null;
+
+        if (_arguments.len == 1) {
+            hint = arguments[0];
+        } else if (_arguments.len > 1) {
+            globalObject.throw("toMatchSnapshot does not support `propertyMatchers` yet.", .{});
```

Comment:
`propertyMatchers` need something like the [Expect Any Constructor](https://jestjs.io/docs/expect#expectanyconstructor) to work correctly.